src/components/TagSet/tagSetChangelogData.ts

```diff
@@ -9,6 +9,15 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Documentation", "Accessibility"],
+    notes: [
+      "Updated `TagSetFilter` to remove button wrapper on each tag when `isDismissible` is false and no `onClick` is passed.",
```

Comment:
Please make a similar change to the language here to match what is in the CHANGELOG.